config/initializers/seeds.rb

```diff
@@ -0,0 +1,21 @@
+return unless (ForemanInternal.table_exists? rescue(false)) && !Foreman.in_rake?
```

Comment:
Should we also return in test env just to make sure it doesn't get seeded by mistake somehow?